providers/common/provider_util.c

```diff
@@ -48,17 +48,14 @@ int ossl_prov_cipher_copy(PROV_CIPHER *dst, const PROV_CIPHER *src)
     return 1;
 }
 
-static int load_common(const OSSL_PARAM params[], const char **propquery,
-                       ENGINE **engine)
+static int load_common(const OSSL_PARAM *propq, const OSSL_PARAM *e,
```

Comment:
Not sure how "common" this function is now. This is handling specific parameters now. Maybe these are just separate functions now?

Reply:
Split and renamed.